dali/operators/reader/loader/video/frames_decoder_test.cc

```diff
@@ -219,13 +219,17 @@ TEST_F(FramesDecoderGpuTest, VariableFrameRate) {
 }
 
 TEST_F(FramesDecoderGpuTest, ConstantFrameRateHevc) {
-  FramesDecoderGpu decoder(cfr_hevc_videos_paths_[0]);
-  RunTest(decoder, cfr_videos_[0]);
+  if (FramesDecoderGpu::SupportsHevc()) {
```

Comment:
```suggestion
  if (!FramesDecoderGpu::SupportsHevc())
    GTEST_SKIP();
```
How about skipping the test so that we don't give the false impression that the test was run?

Reply:
I want to add `else` with assessing the proper exception, when we have it. I just started with this simple thing to alleviate the CI.